build.sh

```diff
@@ -200,7 +200,12 @@ ruby-*)
   if [[ $RUBY = *head* ]]; then
     EXTRA_FLAGS="--rubygems ignore"
   fi
-  announce rvm install $RUBY $EXTRA_FLAGS --verify-downloads 1 $MOVABLE_FLAG --disable-install-doc -C --without-tcl,--without-tk,--without-gmp;;
+
+  if [[ $RUBY = ruby-2.4* ]]; then
+    EXTRA_CONFIGURE_OPTS=",--enable-install-static-library"
+  fi
+
+  announce rvm install $RUBY $EXTRA_FLAGS --verify-downloads 1 $MOVABLE_FLAG --disable-install-doc -C --without-tcl,--without-tk,--without-gmp$EXTRA_CONFIGURE_OPTS;;
```

Comment:
this can be changed to:
```bash
CONFIGURE_OPTS=( --without-tcl --without-tk --without-gmp )
case $RUBY in
  (ruby-2.4*|ruby-2.5*) CONFIGURE_OPTS+=( --enable-install-static-library ) ;;
esac
announce rvm install $RUBY $EXTRA_FLAGS --verify-downloads 1 $MOVABLE_FLAG --disable-install-doc -- ${CONFIGURE_OPTS[@} ;;
```
not sure I didn't do any typo, typed from memory

Reply:
Oh nice, let's prepare for Ruby 2.5 in advance :)